src/index.js

```diff
@@ -0,0 +1,96 @@
+const state = {
+    tempVar : 76,
+    skyDisplay : {
```

Comment:
Nice job storing this configuration data here. See my later comments about how we could make these even more useful!